lattice-control/lattice-control-interface.smithy

```diff
@@ -425,15 +478,17 @@ structure ScaleActorCommand {
 /// A command sent to a host to request that instances of a given actor
 /// be terminated on that host
 structure StopActorCommand {
+    /// The ID of the lattice on which this request will be performed
+    @required    
+    latticeId: String,
+
     /// The ID of the target host
-    @required  
-    @serialization(name: "host_id")  
+    @required      
     hostId: String,
 
     /// Reference for this actor. Can be any of the means of uniquely identifying
     /// an actor
-    @required    
-    @serialization(name:"actor_ref")
+    @required        
```

Comment:
I think this should be actorId as the only means of uniquely identifying an actor that we support is by ID